iphone/cli/commands/_buildModule.js

```diff
@@ -107,7 +138,9 @@ iOSModuleBuilder.prototype.run = function run(logger, config, cli, finished) {
 		'compileJS',
 		'buildModule',
 		'createUniversalBinary',
-		'verifyBuildArch',
+		function (next) {
+			this.verifyBuildArch().then(next, next);
```

Comment:
<!--
  0 failure: 
  2 warning:  iphone/cli/comman..., iphone/cli/comman...
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_buildModule.js;
  Line: 142;
-->

- :warning: iphone/cli/commands/_buildModule.js line 142 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
- :warning: iphone/cli/commands/_buildModule.js line 142 – Expected catch() or return (promise/catch-or-return)